code/game/machinery/recycle_vendor.dm

```diff
@@ -549,15 +589,20 @@
 /obj/machinery/amesilo/Destroy()
 	. = ..()
 	selleverything()
```

Comment:
every material in the AME is exported when it is destroyed, ensuring taxes are paid in the case of the silo being destroyed.